Assets/HoloToolkit/Utilities/Scripts/WorldAnchorManager.cs

```diff
@@ -170,7 +170,7 @@ private void CreateAnchor(GameObject gameObjectToAnchor, string anchorName)
             }
             else
             {
-                // Othertimes we must wait for the 
+                // Othertimes we must wait for the
```

Comment:
Nit: not your change at all, but this is probably meant to say "Other times we must wait for the tracking system to locate the world" or something along those lines.